src/main/java/com/philips/research/spdxbuilder/persistence/blackduck/BlackDuckApi.java

```diff
@@ -247,14 +247,19 @@ public String toString() {
         String licenseType;
         List<LicenseJson> licenses = new ArrayList<>();
 
-        public License getLicense() {
+        public Optional<License> getLicense() {
             if (licenses.isEmpty()) {
                 final var identifier = (spdxId != null) ? spdxId : licenseDisplay;
-                return LicenseParser.parse(identifier);
+                if (identifier.equals("Unknown License")) {
```

Comment:
Are you sure this would be an improvement to the code?

This is the only location where this string occurs, and the constant would obscure the actual name.